index.html

```diff
@@ -217,53 +238,75 @@ <h2>Glossary</h2>
 
 <div class="letter_anchor" id="e">E</div>
 
-<p><dfn data-lt="extended grapheme cluster|extended grapheme clusters" id="def_extended_grapheme_cluster" class="lint-ignore export">Extended grapheme cluster</dfn>. See <a>grapheme cluster</a>.</p>
+<p><dfn class="lint-ignore">European digits</dfn>. A term used by the Unicode Standard to refer to ASCII digits. <a href="https://www.unicode.org/glossary/#european_digits">Unicode definition</a>: <q>Forms of decimal digits first used in Europe and now used worldwide. Historically, these digits were derived from the Arabic digits; they are sometimes called “Arabic numerals,” but this nomenclature leads to confusion with the real Arabic-Indic digits. Also called "Western digits" and "Latin digits." See <a href="https://www.unicode.org/terminology/digits.html"> Terminology for Digits</a> for additional information on terminology related to digits.</q></p>
```

Comment:
perhaps remove "real" from "real Arabic-Indic digits"?

Add data-lt for the alternate terms (western digits, latin digits, etc.)?

Reply:
The 'real' is part of the quoted Unicode definition, so i don't think we can remove it.